data/seedData.py

```diff
@@ -12,84 +12,236 @@
+                                        phone="123-123-1111",
+                                        propertyID=property_meerkat_manor.id,
+                                        staffIDs=[])
+    tenant_starvin_artist.save_to_db()
 
-    newNote = NotesModel(ticketid=0, text="Tenant not responding to phone calls.", user=1)
-    db.session.add(newNote)
-    newNote = NotesModel(ticketid=1, text="Tenant has over 40 cats.", user=2)
-    db.session.add(newNote)
-    newNote = NotesModel(ticketid=1, text="Issue Resolved with phone call", user=3)
-    db.session.add(newNote)
-    newNote = NotesModel(ticketid=2, text="Contacted Tenant -- follow up tomorrow.", user=3)
-    db.session.add(newNote)
+    ticket_roof_on_fire = TicketModel(
+        issue="The roof, the roof, the roof is one fire.",
+        tenant=tenant_renty_mcrenter.id,
+        sender=user_1.id,
+        status="In Progress",
+        urgency="Low",
+        assignedUser=user_mister_sir.id)
+    ticket_roof_on_fire.save_to_db()
+    ticket_dumpster_fire = TicketModel(issue="Flaming Dumpster Fire.",
+                                       tenant=tenant_soho_muless.id,
+                                       sender=user_3.id,
+                                       status="Critical",
+                                       urgency="HIGH",
+                                       assignedUser=user_mister_sir.id)
+    ticket_dumpster_fire.save_to_db()
+    ticket_unpaid_rent = TicketModel(issue="Unpaid Rent",
+                                     tenant=tenant_renty_mcrenter.id,
+                                     sender=user_1.id,
+                                     status="Critical",
+                                     urgency="HIGH",
+                                     assignedUser=user_mister_sir.id)
+    ticket_unpaid_rent.save_to_db()
+    ticket_40_cats = TicketModel(issue="Over 40 cats in domicile.",
+                                 tenant=tenant_soho_muless.id,
+                                 sender=user_3.id,
+                                 status="Critical",
+                                 urgency="HIGH",
+                                 assignedUser=user_mister_sir.id)
+    ticket_40_cats.save_to_db()
 
-    newTicket = TicketModel(issue="The roof, the roof, the roof is one fire.", tenant=1, sender=1, status="In Progress", urgency="Low", assignedUser=4)
-    db.session.add(newTicket)
-    newTicket = TicketModel(issue="Flaming Dumpster Fire.", tenant=2, sender=3, status="Critical", urgency="HIGH", assignedUser=4)
-    db.session.add(newTicket)
-    newTicket = TicketModel(issue="Unpaid Rent", tenant=1, sender=1, status="Critical", urgency="HIGH", assignedUser=4)
-    db.session.add(newTicket)
-    newTicket = TicketModel(issue="Over 40 cats in domicile.", tenant=2, sender=3, status="Critical", urgency="HIGH", assignedUser=4)
-    db.session.add(newTicket)
+    # TODO(@gussmith23) This note doesn't actually attach to a real issue!
```

Comment:
Were these notes to remind us to fix now or later?

Reply:
@gussmith23 If this is an issue and needs to be fixed could you create an new issue instead of adding a TODO comment?

Reply:
It was to remind me to fix later; I will add an issue!

Reply:
Added these as their own issues!